filebeat/docs/faq.asciidoc

```diff
@@ -4,8 +4,26 @@
 This section contains frequently asked questions about Filebeat. Also check out the
 https://discuss.elastic.co/c/beats/filebeat[Filebeat discussion forum].
 
+[[filebeat-network-volumes]]
 === Can I read log files from network volumes?
 
 We do not recommend reading log files from network volumes. Whenever possible, install Filebeat on the host machine and
 send the log files directly from there. Reading files from network volumes (especially on Windows) can have unexpected side
 effects. For example, changed file identifiers may result in Filebeat reading a log file from scratch again.
+
+// === Why isn’t Filebeat collecting lines from my file?
```

Comment:
I'll add this for now and when the code in master is available in a release, we can remove point #2 about the default for ignore_older being 24h.